_posts/0000-01-02-DBM492.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
**Bold** and other *italic* [link to Google!](http://google.com),# This is an <h1> tag, ## This is an <h2> tag, ###### This is an <h6> tag, *This text will be italic*, _This will also be italic_, **This text will be bold**, __This will also be bold__, _You **can** combine them_, ![GitHub Logo](/images/logo.png), Format: ![Alt Text](url), http://github.com - automatic!, [GitHub](http://github.com)
As Kanye West said:
> We're living the future so
> the present is our past.
I think you should use an
`<addr>` element here instead.